subsys/net/l2/openthread/openthread.c

```diff
@@ -401,13 +406,25 @@ static int openthread_init(struct net_if *iface)
 		platformShellInit(ot_context->instance);
 	}
 
-	otIp6SetEnabled(ot_context->instance, true);
+	if (IS_ENABLED(CONFIG_OPENTHREAD_NCP)) {
+		otNcpInit(ot_context->instance);
+	}
+
+	if (IS_ENABLED(CONFIG_OPENTHREAD_RAW)) {
+		otLinkRawSetEnable(ot_context->instance, true);
```

Comment:
This line throws a warning. You need to include the following header:
`#include <openthread/link_raw.h>`

Reply:
Added.